core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -579,6 +582,10 @@ class EMConfig {
             throw ConfigProblemException("The use of 'prematureStop' is meaningful only if the stopping criterion" +
                     " 'stoppingCriterion' is based on time")
         }
+
+        if (namingStrategy == NamingStrategy.DEFAULT) {
```

Comment:
no need for this `DEFAULT`. the default can be directly `NUMBERED`. default is needed when we have differences between configurations, eg when a default  fro black-box testing would be different then a default for white-box testing. but it is not the case here

Reply:
Won't we be using the choice of naming strategies for testing purposes?

Reply:
(moved it to the other config, had misunderstood)